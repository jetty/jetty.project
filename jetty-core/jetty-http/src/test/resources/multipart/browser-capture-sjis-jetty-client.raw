--JettyHttpClientBoundaryny8fndkswj5ot6hx
Content-Disposition: form-data; name="japanese"
Content-Type: text/plain; charset=Shift-JIS

オープンソース
--JettyHttpClientBoundaryny8fndkswj5ot6hx
Content-Disposition: form-data; name="hello"
Content-Type: text/plain; charset=Shift-JIS

日食桟橋
--JettyHttpClientBoundaryny8fndkswj5ot6hx--
