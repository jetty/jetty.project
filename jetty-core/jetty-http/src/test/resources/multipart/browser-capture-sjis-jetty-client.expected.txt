Request-Header|Accept-Encoding|gzip
Request-Header|Connection|close
Request-Header|Content-Type|multipart/form-data; boundary=JettyHttpClientBoundaryny8fndkswj5ot6hx
Request-Header|Host|localhost:9090
Request-Header|Transfer-Encoding|chunked
Request-Header|User-Agent|Jetty/9.4.9.v20180320
Request-Header|X-BrowserId|jetty-client
Parts-Count|2
Part-ContainsContents|japanese|オープンソース
Part-ContainsContents|hello|日食桟橋